contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/test/matchers/ETagHeaderMatcher.java

```diff
@@ -9,22 +9,22 @@
 import org.springframework.test.web.servlet.ResultMatcher;
 
 @RequiredArgsConstructor(access = AccessLevel.PACKAGE)
-public class EtagHeaderMatcher {
+public class ETagHeaderMatcher {
 
-    public static String toEtag(int version) {
+    public static String toETag(int version) {
```

Comment:
`String.valueOf(version)`

Reply:
and if it's a `String` return type, better to just inline that function everywhere

Reply:
Gives me the following exception in tests: `Request processing failed: java.lang.IllegalArgumentException: Could not parse header 'If-Match' with value '0'`, it needs the extra quotes to succeed.

Reply:
Application responds to GET requests with `ETag:""0""`, so in the request matcher also needs the quotes since it calling `equals`